docs/pages/application-access/guides/dynamodb.mdx

```diff
@@ -1,22 +1,24 @@
 ---
-title: Database Access with AWS DynamoDB
-description: How to access AWS DynamoDB with Teleport.
+title: Access AWS DynamoDB as an Application
+description: How to access AWS DynamoDB as an Application with Teleport.
 ---
 
-Access to AWS DynamoDB is provided by [**Teleport Application
+Access to AWS DynamoDB can be provided by [**Teleport Application
```

Comment:
Didn't you just tell me that `can be` should be avoided in docs? :-)

Reply:
Yes I did, and I'm happy that it stuck in your brain! In this case I feel that the two cannot be used interchangeably. This is specifically saying that  what's described below is an option, but not the only one.